tools/cpp/CROSSTOOL

```diff
@@ -896,6 +900,345 @@ toolchain {
   }
 }
 
+toolchain {
+  toolchain_identifier: "vc_14_0_x86_32"
+  host_system_name: "local"
+  target_system_name: "local"
+
+  abi_version: "local"
+  abi_libc_version: "local"
+  target_cpu: "x86_32_windows_msvc"
+  compiler: "cl"
+  target_libc: "msvcrt140"
+  default_python_version: "python2.7"
+  cxx_builtin_include_directory: "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/INCLUDE"
+  cxx_builtin_include_directory: "C:/Program Files (x86)/Windows Kits/10/include/"
+  cxx_builtin_include_directory: "C:/Program Files (x86)/Windows Kits/8.1/include/"
+  cxx_builtin_include_directory: "C:/Program Files (x86)/GnuWin32/include/"
+  cxx_builtin_include_directory: "C:/python_27_amd64/files/include"
```

Comment:
I was wondering why there are hardcoded paths like this in bazel. Turns out it was added to build tensorflow on Windows? Why is that not done in tensoflow, but hardcoded here? Really?

https://github.com/bazelbuild/bazel/commit/e52ebec48b65b8c0664aa208a20f511d83df950a

Reply:
I guess these are fallback paths and they are pretty good.